install/lsstsw.rst

```diff
@@ -18,14 +20,12 @@ If you have issues using lsstsw, here are two ways to get help:
 1. Prerequisites
 ================
 
-The LSST Science Pipelines can generally be compiled on CentOS, Debian, and macOS platforms.
-See :ref:`prereq-platforms` for information about LSST's official reference platform and build reports with other platforms.
+The LSST Science Pipelines are developed and tested primarily on CentOS, but can be compiled and run on macOS, Debian, Ubuntu, and other Linux distributions.
+See :ref:`prereq-platforms` for information about LSST's official reference platform and build reports with other platforms, and follow the instructions under :ref:`system-prereqs` to ensure you have installed the prerequisite software for your platform.
 
 Before you begin:
 
-- Install prerequisites for your platform: :doc:`CentOS / RedHat <prereqs/centos>`, :doc:`Debian / Ubuntu <prereqs/debian>`, or :doc:`macOS <prereqs/macos>`.
 - If you intend to use a Git LFS repository, like `testdata_ci_hsc`_ or `afwdata`_, :doc:`install and configure Git LFS <git-lfs>`.
```

Comment:
git-lfs comes from conda now.

Reply:
In fact, the page linked does account for that, but I agree that the text in the link is misleading. Changed.